pom.xml

```diff
@@ -29,7 +29,7 @@
   <artifactId>wombat</artifactId>
   <name>wombat</name>
   <packaging>war</packaging>
-  <version>3.7.2-SNAPSHOT</version>
+  <version>3.7.4</version>
```

Comment:
@gt-plos What is the correct value for <version>?  is it going to be `3.7.4-SNAPSHOT`?